tests/by-util/test_seq.rs

```diff
@@ -867,7 +867,7 @@ fn test_parse_valid_hexadecimal_float_two_args() {
         (["0x.8p16", "32768"], "32768\n"),
         (["0xffff.4p-4", "4096"], "4095.95\n"),
         (["0xA.A9p-1", "6"], "5.33008\n"),
-        (["0xa.a9p-1", "6"], "5.33008\n"),
+        (["  0xa.a9p-1", "6"], "5.33008\n"),
```

Comment:
This passes without the changes made to `numberparse.rs`. Why not use the input (` 0xee.`) that causes the crash?